app/components/osc-webhook-triggers/osc-webhook-triggers.component.js

```diff
@@ -20,9 +21,57 @@
   });
 
 
-  function OscWebhookTriggers($scope, $uibModal, $filter, APIService) {
+  function OscWebhookTriggers($scope, $uibModal, $filter, $timeout, APIService) {
```

Comment:
We're trying to keep these lists alphabetized since they can get long and hard to maintain